.dockstore.yml

```diff
@@ -344,3 +344,8 @@ workflows:
     primaryDescriptorPath: /pipes/WDL/workflows/trimal.wdl
     testParameterFiles:
       - empty.json
+  - name: 16S_amplicon_analysis
+    subclass: WDL
+    primaryDescriptorPath: /pipes/WDL/workflows/16S_amplicon_analysis.wdl
```

Comment:
See comments below on workflow name, but if anything down there changes, it should change here too. As it stands now, if nothing below changes, this should actually be corrected to `name: qiime_wfl` to match the code below (but ideally it would change to something else).

Reply:
Updated the workflow name to amplicon16S_analysis in latest push